vars/folioSlackNotificationUtils.groovy

```diff
@@ -49,16 +123,15 @@ String renderSlackTestResultMessageSection(TestType type, LinkedHashMap<String,
     , "${url}"
     , useReportPortal
     , ReportPortalTestType.fromType(type).reportPortalLaunchesURL())
-}
+  println("folioSlackNotificationUtils.groovy renderSlackTestResultMessageSection slackTestType.renderSection=${testVal}")
```

Comment:
Does it work in this way?

Reply:
It was left for test purposes. I will delete it before the final merge.

Reply:
Removed.